_layouts/manual.html

```diff
@@ -1,6 +1,7 @@
 ---
 pagename: Manual
 latestversion: "1.22"
+url_suffix: "index"
```

Comment:
Mind naming this "version_selector_target" instead? This feels like some global variable.

Reply:
updated :)